systemd/openqa-livehandler.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Handler for live view in openQA's web UI
+Wants=apache2.service openqa-setup-db.service
+Before=apache2.service
+After=postgresql.service openqa-setup-db.service
+Requires=openqa-resource-allocator.service openqa-scheduler.service openqa-websockets.service
+
+[Service]
+# TODO: define whether we want to run the web ui with the same user
+User=geekotest
+Environment="DBUS_STARTER_BUS_TYPE=system"
```

Comment:
If not mistaken, you don't need dbus for this service

Reply:
Right, I'll remove it.